Sources/PackageGraph/PackageGraph+Loading.swift

```diff
@@ -39,7 +39,7 @@ extension PackageGraph {
         // the URL but that shouldn't be needed after <rdar://problem/33693433>
         // Ensure that identity and package name are the same once we have an
         // API to specify identity in the manifest file
-        let manifestMapSequence = (root.manifests + externalManifests).map({ (PackageIdentity(url: $0.url), $0) })
+        let manifestMapSequence = (root.manifests + externalManifests).map({ (PackageIdentity(url: $0.packageLocation), $0) })
```

Comment:
Not an issue with this PR per se, but this seems wrong to me, root manifests should not be using the `url` variant of identity.

Reply:
agreed that this area needs further cleanup. was trying to be careful not to introduce functional changes in this specific PR, so kept things as-is for now. I have a couple of follow up PRs coming that address this

Reply:
@neonichu In the past, the `url` of a root package was always the same as the `path`, so this would not have had a practical difference in the past.  If the `packageLocation` has the same guarantee for now, then this should have no practical difference, as @tomerd also indicates.

Reply:
Yep, this doesn't have any practical implications right now, but would be good to clean up sooner rather than later since we introduced the two distinct initializers to have a smoother transition to a new identity model.